README.md

```diff
@@ -90,24 +89,28 @@ They are initialized for you and passed to the investigation via investigation.R
 
 ## Testing locally
 
-### Pre-requirements
-- an existing cluster
-- an existing PagerDuty incident for the cluster and alert type that is being tested
-
-To quickly create an incident for a cluster_id, you can run `./test/generate_incident.sh <alertname> <clusterid>`.
-Example usage:`./test/generate_incident.sh ClusterHasGoneMissing 2b94brrrrrrrrrrrrrrrrrrhkaj`.
-
-### Running cadctl for an incident ID
-1) Export the required ENV variables, see [required ENV variables](#required-env-variables).
-2) Create a payload file containing the incident ID
-  ```bash
-  export INCIDENT_ID=
-  echo '{"__pd_metadata":{"incident":{"id":"'${INCIDENT_ID}'"}}}' > ./payload
-  ```
-3) Run `cadctl` using the payload file
-  ```bash
-  ./bin/cadctl investigate --payload-path payload
-  ```
+Requires an existing cluster.
+
+1. Create a test incident and payload file for your cluster
+
+   ```bash
+   ./test/generate_incident.sh <alertname> <clusterid>
+   ```
+
+2. Export the required env variables from vault
+
+   > **Note:** For information on the envs see [required env variables](#required-env-variables).
+
+   ```
+   source test/set_stage_env.sh
```

Comment:
Doesn't this already set the env var, if we move this step before the generate incident, does it fix the issue without needing to touch the generation script?

Reply:
mhh yeah i see what you mean, 

in generate_incident we use the  creds from `osd-sre/configuration-anomaly-detection/cad-testing`
in set_stage_env its stage pd creds  `osd-sre/configuration-anomaly-detection/pd/stg`

it results in kind of the same, so we could change the order 
but I feel like this is good for now, as the incident is usually created only once until you have a new cluster/alert title

setting stage envs is required more often, e.g each time you reopen your terminal/editor